contribution_evaluation.py

```
@@ -0,0 +1,144 @@
+            'patch': patch
+        })
+
+    return changes
+
+# 计算 PR 的工作量评估
+def calculate_pr_workload(changes, owner, repo):
+    workload = 0
+
+    for change in changes:
+        filename = change['filename']
+        patch = change['patch']
```

Ruff (0.8.2)

Local variable `patch` is assigned to but never used

Remove assignment to unused variable `patch`

(F841)